src/test/java/com/puppycrawl/tools/checkstyle/internal/AllChecksTest.java

```diff
@@ -257,7 +257,7 @@ public void testAllChecksWithDefaultConfiguration() throws Exception {
             }
             else {
                 // Checks which have TreeWalker as a parent.
-                BaseCheckTestSupport testSupport = new BaseCheckTestSupport() {
+                final BaseCheckTestSupport testSupport = new BaseCheckTestSupport() {
```

Comment:
@romani @Nimfadora Is this something our check, FinalLocalVariable, missed?
If so, we should make an issue on it.